### PR TITLE
Update ROS contact-us modal

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update ROS contact-us modal

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/robotics#get-in-touch
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1vDbdFBrG6tWDUb5z5ltny_7d-GZU78wS8K7Nru2BSj8/edit#)

### CODE REVIEW

File: templates/shared/forms/interactive/robotics_ros.html [hunk 1]
```diff
@@ -2,52 +2,46 @@
   <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
     <header class="p-modal__header" style="display: block; border-bottom: 0;">
       <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
-      <h2 class="p-modal__title u-sv1" id="modal-title">Let's get your robot rolling</h2>
+      <h2 class="p-modal__title u-sv1" id="modal-title">Let’s get your robot rolling</h2>
     </header>
     <div class="js-pagination js-pagination--1">
-      <p id="modal-description" class="u-no-max-width u-no-margin--bottom">Tell us about your project so we can bring the right team to the conversation.</p>
+      <p id="modal-description" class="u-no-max-width u-no-margin--bottom">Tell us about your project so we can bring the right team to the conversation. If the form doesn't address your questions, please submit your inquiry in the text block below.</p>
```
Comment (GAN-Canonical) [inline templates/shared/forms/interactive/robotics_ros.html:8]: Please add " please press next and submit your inquiry in the text block below.

File: templates/shared/forms/interactive/robotics_ros.html [hunk 2]
```diff
@@ -61,35 +55,39 @@ <h3 class="p-heading--five">How many devices will require ESM coverage yearly?</
...
-            <label for="update-frequency-every-3-months">Every 3 months</label>
+            <input type="radio" id="update-frequency-never" name="update-frequency" value="Never">
+            <label for="update-frequency-never">Never</label>
+            <input type="radio" id="update-frequency-in-real-time" name="update-frequency" value="In real time">
+            <label for="update-frequency-in-real-time">In real time</label>
+            <input type="radio" id="update-frequency-every-6-hours" name="update-frequency" value="Every 6 hours">
+            <label for="update-frequency-every-6-hours">Every 6 hours</label>
+            <input type="radio" id="update-frequency-every-day" name="update-frequency" value="Every day">
+            <label for="update-frequency-every-day">Every day</label>
           </div>
```
Comment (GAN-Canonical) [inline templates/shared/forms/interactive/robotics_ros.html:69]: We have a blank option

File: templates/shared/forms/interactive/robotics_ros.html (hunk 1 above)
Comment (GAN-Canonical) [inline templates/shared/forms/interactive/robotics_ros.html:8]: Please add " please press next and submit your inquiry in the text block below.

**Review by GAN-Canonical — CHANGES_REQUESTED** [review]

File: templates/shared/forms/interactive/robotics_ros.html (hunk 2 above)
Comment (GAN-Canonical) [reply templates/shared/forms/interactive/robotics_ros.html:69]: form

**Review by GAN-Canonical — APPROVED** [review]

**Review by GAN-Canonical — APPROVED** [review]

**Review by GAN-Canonical — APPROVED** [review]

**Review by SirSamTumless — APPROVED** [review]

**Pull request merged**